packages/rum/src/domain/getSessionReplayLink.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { isIE } from '@datadog/browser-core'
+import type { RumConfiguration, ViewContexts } from '@datadog/browser-rum-core'
+import { createRumSessionManagerMock } from '../../../rum-core/test'
+import { getSessionReplayLink } from './getSessionReplayLink'
+import { addRecord, resetReplayStats } from './replayStats'
+
+const DEFAULT_CONFIGURATION = {
+  site: 'datad0g.com',
+} as RumConfiguration
+
+describe('getReplayLink', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Could you add a test for `replay-not-started`?